device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -101,14 +101,9 @@ public MqttIotHubConnection(DeviceClientConfig config) throws IllegalArgumentExc
      *
      * @throws TransportException if a connection could not to be established.
      */
-    public void open(Queue<DeviceClientConfig> deviceClientConfigs) throws TransportException
```

Comment:
:D why did we have a queue if it wasn't allowed to be larger than 1 !!! strange ...